test/index.test.js

```diff
@@ -111,9 +155,92 @@ describe('SSM Helper', () => {
     });
   });
 
-  describe('Setter', () => {
+  describe('getAws', () => {
+    it('successfully gets', (done) => {
+      const testValue = UUID();
+      AWSMock.mock('SSM', 'getParameter', (params, callback) => {
+        callback(null, {
+          Parameter:
+                    {
+                      Name: 'DEV_TESTY',
+                      Type: 'SecureString',
+                      Value: testValue,
+                      Version: 4,
+                      LastModifiedDate: new Date().toISOString(),
+                      ARN: 'arn:aws:ssm:eu-west-1:*:parameter/DEV_TESTY',
+                    },
+        });
+      });
+
+      const SSM = new SSMHelper({});
+      SSM.getAws('DEVB_TESTY')
+        .then((result) => {
+          expect(result).to.eql(testValue);
+          done();
+        })
+        .catch(done);
+    });
+    it('successfully gets and adds to cache', (done) => {
+      const testValue = UUID();
+      AWSMock.mock('SSM', 'getParameter', (params, callback) => {
+        callback(null, {
+          Parameter:
+                    {
+                      Name: 'DEV_TESTY',
+                      Type: 'SecureString',
+                      Value: testValue,
+                      Version: 4,
+                      LastModifiedDate: new Date().toISOString(),
+                      ARN: 'arn:aws:ssm:eu-west-1:*:parameter/DEV_TESTY',
+                    },
+        });
+      });
+
+      const SSM = new SSMHelper({
+        cache: true,
+      });
+      SSM.getAws('DEV_TESTY')
+        .then((result) => {
+          expect(result).to.eql(testValue);
```

Comment:
Worth checking it's actually in the cache? `expect(SSM.cache.get('DEV_TESTY')).to.eql(testValue)`